### DISCUSSION TITLE
Settings json not being applied

### DISCUSSION THREAD

**Discussion by havingpun** [UltimaHoarder/UltimaScraper#1496] [Q&A]
Hello all,

I am having trouble getting settings to be applied. For example I did the Blacklist, date format, and filename format changes, but none of them seem to work when I run the script. Any ideas on what I'm doing wrong?

**Comment by fg4jerem** [comment]
Sounds like a case where the changes you applied weren't to the correct files, or in the wrong section of the file. Be careful between settings for "onlyfans" and settings for "starsavn". I got bitten by such a mistake many times.

**Reply by havingpun** [reply]
You were right! I did it in the wrong section. However I now have the filename format correct, but the blacklist still doesn't work..

**Reply by LunarPenguin** [reply]
please update to the latest version and try again.
